docs/source/quickstart.md

```diff
@@ -107,34 +108,32 @@ Generate a default config file:
 
     jupyterhub --generate-config
 
-#### Customize the configuration, authentication, and process spawning
+#### Customize the configuration, authentication, and spawner
 
-Spawn the server on ``10.0.1.2:443`` with **https**:
+For example, to start the jupyterhub on ``10.0.1.2:443`` with **https**:
 
     jupyterhub --ip 10.0.1.2 --port 443 --ssl-key my_ssl.key --ssl-cert my_ssl.cert
 
-The authentication and process spawning mechanisms can be replaced,
-which should allow plugging into a variety of authentication or process
-control environments. Some examples, meant as illustration and testing of this
-concept, are:
+The authenticator and spawner can be replaced from the default provided.
```

Comment:
awkward phrasing "replaced from the default". Perhaps "The default authenticator...can be replaced"?